### PR TITLE
feat: simplify the description in the Hacktoberfest demo

### PR DESCRIPTION
**Pull request by tbouffard**
Use shorter text and reduce the margins to leave more space for the demonstration itself.

covers #456

**Live environment of examples**

https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/feat/hacktoberfest_demo_description/examples/index.html

### CODE REVIEW

File: demo/hacktoberfest-custom-themes/index.html [hunk 1]
```diff
@@ -72,11 +72,10 @@
...
             <h2>Hacktoberfest diagram</h2>
-            <p>You can use this diagram to promote your project in order to have contributions during Hacktoberfest:
-            <ul>
-                <li>Enter your project name</li>
-                <li>Take a screenshot of your favorite diagram theme</li>
-                <li>Advertise about your project with the diagram (Don't forget to add a link to this page)</li>
+            <p class="mb-0">You can use this diagram to promote your project in order to have contributions during Hacktoberfest:
+            <ul class="mt-0">
+                <li>Make it your own: change the project name to yours and choose your favorite theme</li>
+                <li>Share: take a screenshot and advertise about your project (don't forget to add a link to this page)</li>
```
Comment (tbouffard) [inline demo/hacktoberfest-custom-themes/index.html:78]: **suggestion:** alternate text
don't forget to add a link to this page --> don't forget to credit this page

Comment (csouchet) [reply demo/hacktoberfest-custom-themes/index.html:78]: I prefer the suggestion 😊

Comment (tbouffard) [inline demo/hacktoberfest-custom-themes/index.html:78]:
```suggestion
                <li>Share: take a screenshot and advertise about your project (don't forget to credit this page)</li>
```

Comment (tbouffard) [reply demo/hacktoberfest-custom-themes/index.html]: done in d0676390e71de11857a7ad4806782c44a46e2fa5

**Review by csouchet — APPROVED** [review]

**Pull request merged**